.github/workflows/remove-labels.yml

```diff
@@ -15,4 +15,4 @@ jobs:
       - name: Remove labels
         uses: andymckay/labeler@e6c4322d0397f3240f0e7e30a33b5c5df2d39e90
         with:
-          remove-labels: triage, needs-more-info
\ No newline at end of file
+          remove-labels: triage, needs-more-info
```

Comment:
I don't mind either way, but except for `auto_close_prs.yml` it looks like all these workflow changes are just whitespace.

Reply:
Yeah it's whitespace changes, but it ensures proper formatting so I'll leave it.